2020-04-17 01:34:59,098 INFO    MainThread:13780 [wandb_setup.py:_log_setup():205] Logging to wandb\wandb-20200417_013459-13780\debug.log
2020-04-17 01:34:59,098 INFO    MainThread:13780 [wandb_setup.py:_setup():231] multiprocessing start_methods=spawn
2020-04-17 01:34:59,113 INFO    MainThread:13780 [wandb_init.py:init():303] runconfig: {'_wandb': {'desc': None, 'value': {'cli_version': '0.0.14'}}}
2020-04-17 01:35:01,794 INFO    MainThread:13780 [wandb_init.py:init():323] atexit reg
2020-04-17 01:35:01,794 INFO    MainThread:13780 [wandb_init.py:_redirect():177] redirect
2020-04-17 01:35:01,794 INFO    MainThread:13780 [redirect.py:install():72] install start
2020-04-17 01:35:01,795 INFO    MainThread:13780 [redirect.py:install():87] install stop
2020-04-17 01:35:01,804 INFO    MainThread:13780 [redirect.py:install():72] install start
2020-04-17 01:35:01,804 INFO    MainThread:13780 [redirect.py:install():87] install stop
2020-04-17 01:35:01,805 INFO    MainThread:13780 [wandb_init.py:_redirect():204] redirect2
2020-04-17 01:35:01,806 INFO    MainThread:13780 [wandb_run.py:save():186] Saving from wandb-jobspec.json to wandb\wandb-20200417_013459-13780\files\wandb-jobspec.json
2020-04-17 01:35:01,812 INFO    stdout    :13780 [wandb_init.py:_callback():173] callback: stdout, b'\x1b[H\x1b[2J'
2020-04-17 01:35:01,814 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'2020-04-17 01:35:01,813 | INFO: Params file loaded successfully. | (__main__.utils.utils)\r\n'
2020-04-17 01:35:01,818 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'2020-04-17 01:35:01,814 | INFO: Paths initialized successfully. | (__main__.utils.utils)\r\n'
2020-04-17 01:35:01,844 INFO    stdout    :13780 [wandb_init.py:_callback():173] callback: stdout, b'Files already downloaded and verified'
2020-04-17 01:35:01,844 INFO    stdout    :13780 [wandb_init.py:_callback():173] callback: stdout, b'\r\n'
2020-04-17 01:35:04,004 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'Traceback (most recent call last):\r\n'
2020-04-17 01:35:04,005 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'  File "train.py", line 260, in <module>\r\n'
2020-04-17 01:35:04,005 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'    '
2020-04-17 01:35:04,006 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'main()\r\n  File "train.py", line 242, in main\r\n'
2020-04-17 01:35:04,007 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'    '
2020-04-17 01:35:04,007 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'dataloader = make_dataset(params)\r\n  File "train.py", line 60, in make_dataset\r\n'
2020-04-17 01:35:04,008 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'    '
2020-04-17 01:35:04,008 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'drop_last=True)\r\n  File "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\torch\\utils\\data\\dataloader.py", line 219, in __init__\r\n'
2020-04-17 01:35:04,009 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'    '
2020-04-17 01:35:04,010 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'batch_sampler = BatchSampler(sampler, batch_size, drop_last)\r\n  File "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\torch\\utils\\data\\sampler.py", line 190, in __init__\r\n'
2020-04-17 01:35:04,011 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'    "but got batch_size={}".format(batch_size))\r\nValueError: batch_size should be a positive integer value, but got batch_size=[20, 32, 64, 128]'
2020-04-17 01:35:04,012 INFO    stderr    :13780 [wandb_init.py:_callback():173] callback: stderr, b'\r\n'
2020-04-17 01:35:04,015 INFO    MainThread:13780 [wandb_init.py:_atexit_cleanup():163] got exit ret: exit {
}
control {
  req_resp: true
}

2020-04-17 01:35:04,016 INFO    MainThread:13780 [wandb_init.py:_restore():226] restore
2020-04-17 01:35:04,016 INFO    MainThread:13780 [redirect.py:uninstall():90] uninstall start
2020-04-17 01:35:04,016 INFO    MainThread:13780 [redirect.py:_stop():130] _stop: stdout
2020-04-17 01:35:04,017 INFO    stdout    :13780 [redirect.py:_pipe_relay():41] relay done saw last write: stdout
2020-04-17 01:35:04,017 INFO    MainThread:13780 [redirect.py:_stop():136] _stop closed: stdout
2020-04-17 01:35:04,017 INFO    stdout    :13780 [redirect.py:_pipe_relay():47] relay done done: stdout
2020-04-17 01:35:04,018 INFO    MainThread:13780 [redirect.py:_stop():142] _stop joined: stdout
2020-04-17 01:35:04,018 INFO    MainThread:13780 [redirect.py:_stop():144] _stop rd closed: stdout
2020-04-17 01:35:04,018 INFO    MainThread:13780 [redirect.py:uninstall():93] uninstall done
2020-04-17 01:35:04,018 INFO    MainThread:13780 [redirect.py:uninstall():90] uninstall start
2020-04-17 01:35:04,018 INFO    MainThread:13780 [redirect.py:_stop():130] _stop: stderr
2020-04-17 01:35:04,018 INFO    stderr    :13780 [redirect.py:_pipe_relay():41] relay done saw last write: stderr
2020-04-17 01:35:04,019 INFO    MainThread:13780 [redirect.py:_stop():136] _stop closed: stderr
2020-04-17 01:35:04,019 INFO    stderr    :13780 [redirect.py:_pipe_relay():47] relay done done: stderr
2020-04-17 01:35:04,019 INFO    MainThread:13780 [redirect.py:_stop():142] _stop joined: stderr
2020-04-17 01:35:04,019 INFO    MainThread:13780 [redirect.py:_stop():144] _stop rd closed: stderr
2020-04-17 01:35:04,019 INFO    MainThread:13780 [redirect.py:uninstall():93] uninstall done
